store/resource.go

```diff
@@ -0,0 +1,19 @@
+package store
```

Comment:
@rohilsurana this file can be called store only as we will put all repo interfaces on this one.